### PR TITLE
rename form

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/71
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/66
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1185

**Comment by Thiblef** [comment]
:+1:

File: GroupBundle/Form/Type/GroupDocumentType.php [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use OpenOrchestra\BackofficeBundle\Model\GroupInterface;
 use OpenOrchestra\Backoffice\Manager\TranslationChoiceManager;
 
 /**
- * Class OrchestraGroupType
+ * Class GroupDocumentType
  */
-class OrchestraGroupType extends AbstractOrchestraGroupType
+class GroupDocumentType extends AbstractType
```
Comment (nicolasThal) [inline GroupBundle/Form/Type/GroupDocumentType.php]: I think that you forgot to rename the service parameter !!!

File: BackofficeBundle/Form/Type/AbstractOrchestraGroupType.php [hunk 2]
```diff
@@ -1,19 +0,0 @@
-<?php
```
Comment (nicolasThal) [inline BackofficeBundle/Form/Type/AbstractOrchestraGroupType.php]: You should not have suppressed this class !! it was mandatory for the group choice form type

**Pull request merged**